brainscore_core/plugin_management/parse_plugin_changes.py

```diff
@@ -140,12 +171,20 @@ def run_changed_plugin_tests(changed_files: str, domain_root: str):
 
 	if plugin_info_dict["modifies_plugins"]:
 		tests_to_run = []
-		for plugin_type in plugin_info_dict["changed_plugins"]:
-			changed_plugins = plugin_info_dict["changed_plugins"][plugin_type]
-			for plugin_dirname in changed_plugins:
-				root = Path(f'{domain_root}/{plugin_type}/{plugin_dirname}')
-				for filepath in root.rglob(r'test*.py'):
-					tests_to_run.append(str(filepath))
+		for plugin_type in PLUGIN_DIRS:
+			if plugin_type in plugin_info_dict["test_all_plugins"]:
+				plugin_type_dir = Path(f'{domain_root}/{plugin_type}')
+				for dir in plugin_type_dir.iterdir():
+					if dir.is_dir():
+						for filepath in dir.rglob(r'test*.py'):
+							tests_to_run.append(str(filepath))
```

Comment:
there's too many indentations imo and the best would be to refactor and unify, but here's a less pretty "fix" to remove one indent in case
```suggestion
						tests_to_run += [str(filepath) for filepath in dir.rglob(r'test*.py')]
```